backend/plugin/db/mysql/mysql.go

```diff
@@ -26,6 +26,11 @@ import (
 	v1pb "github.com/bytebase/bytebase/proto/generated-go/v1"
 )
 
+const (
+	chunkedSubmissionMaximumSize = 500 * 1024
```

Comment:
Use common.MaxSheetCheckSize instead. One place for all hacks. Also move maxChunksCount there.